src/Directory.Build.props

```diff
@@ -16,6 +16,7 @@
     <DebugType>Full</DebugType>
     <DebugSymbols>true</DebugSymbols>
     <LangVersion>9.0</LangVersion>
+    <WarningLevel>6</WarningLevel>
```

Comment:
><WarningLevel>6</WarningLevel> [](http://example.com/codeflow?start=4&length=30)

To get more consistent error messages between Debug and Release

Reply:
When you say Debug and Release here, do you mean PR build and Main build?

Reply:
At some point of my testing, the warning level was different between Debug and Release configurations, hence this "fix"